README.md

```diff
@@ -180,7 +180,7 @@ batch(() => {
 Batches can be nested and updates will be flushed when the outermost batch call completes.
 
 ```js
-import { signal, computed, effect, batch } from "@preact/signals-core";
+import { signal, computed, effect batch } from "@preact/signals-core";
```

Comment:
```suggestion
import { signal, computed, effect, batch } from "@preact/signals-core";
```

Reply:
Yeah, bad timing there :D I'm trying to rebase the PR using the web UI since I'm not on my main dev machine at the moment. You were too fast with merging the other PR! :D